lib/Doctrine/ODM/MongoDB/Iterator/PrimingIterator.php

```diff
@@ -73,6 +78,10 @@ public function next(): void
         $this->iterator->next();
     }
 
+    /**
+     * @return TKey
```

Comment:
```suggestion
     * @return TKey|null
```

`key()` returns `null` on failure.